asn1/InterledgerPaymentRequest.asn

```diff
@@ -1,21 +1,27 @@
-InterledgerProtocol
+InterledgerPaymentRequest
 DEFINITIONS
 AUTOMATIC TAGS ::=
 BEGIN
 
 IMPORTS
-    UInt8
+    UInt8,
     UInt256
     FROM GenericTypes
+
+    InterledgerPacket
+    FROM InterledgerPacket
 ;
 
 InterledgerPaymentRequest ::= SEQUENCE {
     -- IPR Version
     version UInt8,
     -- Cryptographic condition
     condition UInt256,
-    -- ILP Payment Packet
-    packet OCTET STRING
+    -- please note, InterledgerProtocolPayment
+    -- should not be added directly, it should
+    -- be inside its
+    -- InterledgerPacket envelope
```

Comment:
I don't think this comment is necessary. The ILP packet _is_ the `InterledgerProtocolPayment` inside the packet wrapper. Interledger packets should never appear without that wrapper with the type and the length. Is there somewhere that isn't clear?